projects/opendr_ws/src/perception/scripts/skeleton_based_action_recognition.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python
+# Copyright 2020-2021 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import rospy
+import torch
+import numpy as np
+import pandas as pd
+from std_msgs.msg import String
+from vision_msgs.msg import ObjectHypothesis
+from vision_msgs.msg import Detection2DArray
+from sensor_msgs.msg import Image as ROS_Image
+from opendr_bridge import ROSBridge
+from opendr.perception.pose_estimation import draw
+from opendr.perception.pose_estimation import LightweightOpenPoseLearner
+from opendr.perception.skeleton_based_action_recognition import SpatioTemporalGCNLearner
+from opendr.perception.skeleton_based_action_recognition import ProgressiveSpatioTemporalGCNLearner
+
+NTU60_ClASSES = pd.read_csv(
```

Comment:
Does this work when the ros node is executed from the opendr_ws folder?

Reply:
This is removed from the implementation and added to the dataset part of the learners. So no need to load the dictionary anymore.